src/js/typography.js

```diff
@@ -0,0 +1,119 @@
+const FontFaceObserver = require('fontfaceobserver/fontfaceobserver.standalone.js');
+const superstore = require('superstore-sync');
+
+class Typography {
+
+	constructor (typographyEl, opts) {
+		this.typographyEl = typographyEl;
+
+		this.opts = opts || Typography.getOptions(typographyEl);
+		this.opts = Typography.checkOptions(this.opts);
+
+		this.fontConfigs = [
+			{
+				family: 'FinancierDisplayWeb',
+				weight: 'normal',
+				label: 'serifDisplay'
+			},
+			{
+				family: 'MetricWeb',
+				weight: 'normal',
+				label: 'sans'
+			},
+			{
+				family: 'MetricWeb',
+				weight: 600,
+				label: 'sansBold'
+			},
+			{
+				family: 'FinancierDisplayWeb',
+				weight: 700,
+				label: 'serifDisplayBold'
+			}
+		];
+
+		this.loadFonts();
```

Comment:
I've been thinking along the lines of build service users who would probably want to just autoload the components and have them work as expected. Maybe we could specify a flag for non-build service users to prevent it from autoloading? @wheresrhys @i-like-robots ?